Add API versioning 2019-04.

stevenbrookes opened the pull request:
Hi Tareq

As discussed see what you think of this.  We are still running some testing but looks good so far,.

Regards

Steve

shawnhind commented:
I think the API versioning should be set at the config level.  It's possible that for many API version updates that the library will not need to be updated but that the data that gets returned could change.  

Perhaps specify versions of the API that have been tested with the library and default to 2019-04 but allow setting version through the config.

tareqtms commented:
@stevenbrookes 
I have implemented it in a different way. Added it in `$config` Array, key `ApiVersion`. So it can be customized by users through `$config` when needed. 
Please check commit [a1fcdec](https://github.com/phpclassic/php-shopify/commit/a1fcdecef6ac55ffa4971cb690bfc1fa9c0e2c83)

stevenbrookes commented:
Hi Tareq

 

Cool that seems like a fine way to do it.  Cheers

 

Steve

 

From: Tareq Mahmood <notifications@github.com> 
Sent: 24 May 2019 08:33
To: phpclassic/php-shopify <php-shopify@noreply.github.com>
Cc: Steven Brookes <steven.brookes@talisman-innovations.com>; Mention <mention@noreply.github.com>
Subject: Re: [phpclassic/php-shopify] Add API versioning 2019-04. (#80)

 

@stevenbrookes <https://github.com/stevenbrookes> 
I have implemented it in a different way. Added it in $config Array, key ApiVersion. So it can be customized by users through $config when needed.
Please check commit a1fcdec <https://github.com/phpclassic/php-shopify/commit/a1fcdecef6ac55ffa4971cb690bfc1fa9c0e2c83> 

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/phpclassic/php-shopify/pull/80?email_source=notifications&email_token=AKJVX3IP4VSAUTSR4HPDWP3PW6K4DA5CNFSM4HJOJP2KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODWEM3JI#issuecomment-495504805> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AKJVX3I47XJL6MVSL5ZGLZDPW6K4DANCNFSM4HJOJP2A> .  <https://github.com/notifications/beacon/AKJVX3NE64ZGQBQYLHEHQL3PW6K4DA5CNFSM4HJOJP2KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODWEM3JI.gif> 



---
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus
